### PR TITLE
Implement the direct message page

### PR DESCRIPTION
**Pull request by MidfieId**
added the Direct Messages Page as per the design.

made changes to the app.css files accordingly to style the page.

made the TextMessage.cs file as a model for the text messages to be typed in the chat box, this can be later on used for the main page chat box as well when implemented,

### CODE REVIEW

File: Frontend/Components/App.razor [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <base href="/" />
     <link rel="stylesheet" href="app.css" />
-    <link rel="stylesheet" href="LDEV.Frontend.styles.css" />
```
Comment (ahmed-okkio) [inline Frontend/Components/App.razor:9]: Why did this get removed?

File: Frontend/Components/App.razor [hunk 2]
```diff
@@ -16,7 +16,7 @@
 </head>
 
 <body>
-    <Routes />
+    <Routes  />
```
Comment (ahmed-okkio) [inline Frontend/Components/App.razor]: what zis

File: Frontend/Components/Pages/LoginPage.razor [hunk 3]
```diff
@@ -26,6 +26,7 @@
 
 @code
 {
+
```
Comment (ahmed-okkio) [inline Frontend/Components/Pages/LoginPage.razor]: This whole file has accidental changes

File: Frontend/Components/_Imports.razor [hunk 4]
```diff
@@ -8,3 +8,5 @@
 @using Microsoft.JSInterop
 @using Frontend
 @using Frontend.Components
+@using Microsoft.AspNetCore.Authorization
```
Comment (ahmed-okkio) [inline Frontend/Components/_Imports.razor:11]: Unused imports?

File: Frontend/Models/TextMessage.cs [hunk 5]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Frontend.Models
```
Comment (ahmed-okkio) [inline Frontend/Models/TextMessage.cs]: Should probably reference the original textmessage class from backend, should  be in Backend.Models

File: Frontend/Program.cs [hunk 6]
```diff
@@ -5,6 +5,9 @@
 // Add services to the container.
 builder.Services.AddRazorComponents();
 
+
```
Comment (ahmed-okkio) [inline Frontend/Program.cs]: This whole file has accidental changes

File: Frontend/Components/Pages/DmPage.razor [hunk 7]
```diff
@@ -1,3 +1,32 @@
 @page "/dm"
+@using Models
+@inject NavigationManager navMan
 
-<h3>this is the DM page</h3>
\ No newline at end of file
+<button class="dmChatButton" @onclick="backToMain"> Main Chat </button>
```
Comment (ahmed-okkio) [inline Frontend/Components/Pages/DmPage.razor:5]: this button doesn't seem to work

File: Frontend/Components/Pages/DmPage.razor [hunk 8]
```diff
@@ -1,3 +1,32 @@
...
+<div class="chatMessageDiv">
```
Comment (ahmed-okkio) [inline Frontend/Components/Pages/DmPage.razor:6]: Could we make the area that will contain the text message a different shading?
For example discord uses a different shade for each section:
image

File: Frontend/Components/App.razor (hunk 1 above)
Comment (MidfieId) [reply Frontend/Components/App.razor:9]: this was a left over css file from the boiler plate code, it serves no purpose anymore.

File: Frontend/Components/Pages/DmPage.razor (hunk 8 above)
Comment (MidfieId) [reply Frontend/Components/Pages/DmPage.razor:6]: A different shading has been made for the message area.

**Pull request closed without merging**